src/main/java/com/puppycrawl/tools/checkstyle/utils/CheckUtil.java

```diff
@@ -589,4 +589,16 @@ private static int lastIndexOfNonWhitespace(String line) {
         }
         return length;
     }
+
+    /**
+     * Checks if a token type is a literal true or false.
+     *
+     * @param tokenType the TokenType
+     * @return true if tokenType is LITERAL_TRUE or LITERAL_FALSE
+     */
+    public static boolean isBooleanLiteralType(final int tokenType) {
```

Comment:
Why `CheckUtil`? We have `TokenUtil` for such methods. Please move.

Reply:
Done.